src/test/java/org/example/homework13/StringUtilsTests/StringUtilsTests.java

```diff
@@ -0,0 +1,113 @@
+package org.example.homework13.StringUtilsTests;
+
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class StringUtilsTests {
+    @Test
+    @Tag("1st")
+    @DisplayName("Check user's email basic correctness")
+    void checkEmail() {
```

Comment:
actually the idea was to test StringUtils methods, not the email. Looking to your test it is not obvious, that you are checking StringUtils method